api/src/services/__tests__/programsService.ts

```diff
@@ -8,6 +8,7 @@ import {
   listActivityTypes,
   findProgramWithActivities,
   listProgramsForCurriculum,
+  checkForPrefill,
```

Comment:
Nice job importing the new data in the file and fixing all the data that was previously there.